stix_shifter/stix_transmission/src/modules/splunk/splunk_error_mapper.py

```diff
@@ -0,0 +1,33 @@
+from .....utils.error_mapper_base import ErrorMapperBase
+from .....utils.error_response import ErrorCode
+
+error_mapping = {
+        'Unknown sid.': ErrorCode.TRANSMISSION_SEARCH_DOES_NOT_EXISTS,
```

Comment:
Could you make these a constant defined in ErrorCode?
`Unknown sid.` and `Unable to parse the search`

Reply:
It is a module specific stuff. These lines are used in the splunk  module only.